core/consensus/src/wal.rs

```diff
@@ -63,6 +68,26 @@ impl SignedTxsWAL {
         Ok(())
     }
 
+    pub fn available_height(&self) -> ProtocolResult<Vec<u64>> {
```

Comment:
What does this function use？I have not seen it.

Reply:
oh~,that's for cli maintainance to list all wal file for their heights and help to remove wal files.
That's could be only used in cli maintainance tool.

Reply:
You should write a comment on this function for other developers to know what it does.